lib/fastlane/plugin/wpmreleasetoolkit/actions/bundletool/bundletool_generate_universal_signed_apk_action.rb

```diff
@@ -0,0 +1,104 @@
+module Fastlane
+  module Actions
+    class BundletoolGenerateUniversalSignedApkAction < Action
+      def self.run(params)
+        aab_path = params[:aab_path]
+        apk_output_path = params[:apk_output_path]
+        keystore_path = params[:keystore_path]
+        keystore_password = params[:keystore_password]
+        keystore_key_alias = params[:keystore_key_alias]
+        signing_key_password = params[:signing_key_password]
+
+        begin
+          sh('command -v bundletool > /dev/null')
+        rescue StandardError
+          UI.user_error!('bundletool is not installed. Please install it using the instructions at https://developer.android.com/studio/command-line/bundletool')
+          raise
+        end
+
+        sh('bundletool', 'build-apks',
+           '--mode', 'universal',
+           '--bundle', aab_path,
+           '--output-format', 'DIRECTORY',
+           '--output', apk_output_path,
+           '--ks', keystore_path,
+           '--ks-pass', keystore_password,
+           '--ks-key-alias', keystore_key_alias,
+           '--key-pass', signing_key_password)
```

Comment:
It would be nice if those four code-signing-related parameters were optional, so that it would also be possible to generate non-signed APKs as well as signed APKs depending on the needs (and if we have the signing keys at all).

Reply:
Good call! I was thinking of a use case where we would want to generate a non-signed APK but didn't think of apps using Google's Managed Code Signing. So yes, making these optional makes a lot of sense.

Reply:
Addressed in 16e5050557dcad2312d1b7db19c7b1ddfa71119b